packages/indicators/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Had to add `tsconfig` file for the `indicators` package because after including test files in the TS build was failing.